docs/Model_Assumptions.rst

```diff
@@ -63,6 +63,61 @@ In case of excessive excess energy, a warning is given that it seems to be cheap
 High excess energy can for example result into an optimized inverter capacity that is smaller than the peak generation of installed PV.
 This becomes unrealistic when the excess is very high.
 
+Stratified thermal storage
```

Comment:
Can you change this to `Energy storage` and descibe how the generic energy storage is used, and then `stratified thermal storage` as a specific case of energy storages?

Outline of content:
Generic storages are defined with file `x` and `y`, and have subassets a, b, c. Self-discharge... efficiency of charge/discharge non-dependent on soc... rate at which battery can be (dis)charged relative to its capacity (crate)... battery lifetime based on years, not charge cycles... for now now KPI measuring the storage utilization (charge cycles, average charge ect).
Stratified storage uses two optional parameters: d, e. If those two are not included in the csv file (or 0), then a normal generic storage is simulated.

Reply:
> Generic storages are defined with file `x` and `y`, and have subassets a, b, c. Self-discharge... efficiency of charge/discharge non-dependent on soc... rate at which battery can be (dis)charged relative to its capacity (crate)... battery lifetime based on years, not charge cycles... for now now KPI measuring the storage utilization (charge cycles, average charge ect).

I don't quite understand. Isn't this section described in `Parameters and Definitions in CSVs/JSON` in the docs? I can list the subassets but do I need to describe them again or can I just reference to `Parameters and Definitions in CSVs/JSON`? And should I actually add the fixed thermal losses to `MVS_parameters.rst` as well or did you not want them in there?

Reply:
Is there a reason why you don't use references to other rst files in rtd, or did you just not have a need for it until now?

Otherwise I would suggest this:

- Adding `sphinx.ext.autosectionlabel` to the `extensions` in `docs/config.py`
- Adding a label to the subsection `storage_*.csv` in `docs/MVS_parameters.rst`
- Adding the following sentence to docs/Model_Assumptions.rst:
   ```Generic storages are defined with file `energyStorage.csv` and `storage_*.csv` and have subassets, which are listed in :ref:`storage_*.csv`.```

I've tested it locally and it works.

Reply:
So I've found a solution in commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/ad4ef158db52c5ebaba8826fe3c94fe853117839

What do you think @smartie2076? Is it necessary to describe the generic storage in more detail?